spec/rubocop/cop/style/collection_compact_spec.rb

```diff
@@ -114,6 +114,24 @@
         array.lazy.compact!
       RUBY
     end
+
+    it 'registers an offense and corrects when using `to_enum.reject` on array to reject nils and receiver is a variable' do
+      expect_offense(<<~RUBY)
+        def foo(params)
+          array.to_enum.reject { |e| e.nil? }
```

Comment:
In this example the direct receiver is `to_enum`, not a variable, so I find the description to be a bit misleading.

Reply:
Oops... actually the repro code is also wrong. I've updated this PR.